api/Business/BusinessObjects/UserBusinessObject.cs

```diff
@@ -48,6 +50,19 @@ public async Task<OperationResult> Update(Guid uuid, User record)
             });
         }
 
+        public async Task<OperationResult<List<User>>> ListFilteredUsers(string search, int sort)
+        {
+            return await ExecuteOperation(async () => {
+
+                if (sort !=0 && sort != 1)
+                {
+                    throw new Exception();
+                }
+                var result =await _userDataAccessObject.FilterUsers(search,sort);
```

Comment:
don't use "var" use the real type: for this case I think it's List<User>

Reply:
donw